Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,19 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Save Encodable object to a given URL.
+    ///
+    /// - Parameters:
+    ///   - url: URL to save object to.
+    /// - Throws: Throws any errors thrown by JSONEncoder or FileManager.
+    public static func encode<T: Encodable>(_ object: T, to url: URL) throws {
```

Comment:
@AlexeiGitH What do you think about adding a JSONEncoder parameter with default value JSONEncoder() so people can use custom encoders here?